description: fix path to freebsd's kernel headers (not working yet)
author: Michael Gilbert <mgilbert@debian.org>

--- wine-1.6.1.orig/dlls/ntdll/signal_x86_64.c
+++ wine-1.6.1/dlls/ntdll/signal_x86_64.c
@@ -164,7 +164,7 @@ extern int arch_prctl(int func, void *pt
 #define FPU_sig(context)     ((XMM_SAVE_AREA32 *)((context)->uc_mcontext.fpregs))
 
 #elif defined(__FreeBSD__) || defined (__FreeBSD_kernel__)
-#include <sys/ucontext.h>
+#include <machine/ucontext.h>
 
 #define RAX_sig(context)     ((context)->uc_mcontext.mc_rax)
 #define RBX_sig(context)     ((context)->uc_mcontext.mc_rbx)
